src/gui/presets.cpp

```diff
@@ -562,24 +579,26 @@ void FurnaceGUI::initSystemPresets() {
       DIV_SYSTEM_OPN_EXT, 64, 0, 2, // 4MHz
       DIV_SYSTEM_PCM_DAC, 64, 0, 44099|(15<<16)|(1<<20),
       DIV_SYSTEM_OPL3_DRUMS, 64, 0, 2,
+      DIV_SYSTEM_PCSPKR, 64, 0, 0,
       0
     }
   ));
   cat.systems.push_back(FurnaceGUISysDef(
     "ZX Spectrum (48K)", {
-      DIV_SYSTEM_AY8910, 64, 0, 2,
       DIV_SYSTEM_SFX_BEEPER, 64, 0, 0,
       0
     }
   ));
   cat.systems.push_back(FurnaceGUISysDef(
     "ZX Spectrum (128K)", {
+      DIV_SYSTEM_SFX_BEEPER, 64, 0, 0,
```

Comment:
Best don't - consider hardware playback. Only AY or beeper, but not both.

Reply:
The thing is - most of the presets dont consider hardware playback at all. See: C64 Sound Expander, Family Noraebang, most, if not all arcade presets and many MSX presets (ZX too!).

I understand what you do mean, but many people felt confused, thinking 128K Spectrum has no beeper. I may change that, if you _really_ insist.

Reply:
ZX beeper requires precise timing, which would be very difficult to do if having to drive AY as well.

Reply:
Another option is to add a new preset called ZX Spectrum 128K (with beeper)...

Reply:
128k preset with beeper might be a better choice (but then, what about all other ZX+enhancement presets?)

Reply:
Don't worry about them for now.

Reply:
removed beeper from 128k :man_shrugging: